buildpipeline/Core-Setup-Publish.json

```diff
@@ -22,6 +40,103 @@
     },
     {
       "enabled": true,
+      "continueOnError": true,
+      "alwaysRun": false,
+      "displayName": "Cleanup previous tools source if present",
+      "timeoutInMinutes": 0,
+      "task": {
+        "id": "e213ff0f-5d5c-4791-802d-52ea3e7be1f1",
+        "versionSpec": "1.*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "scriptType": "inlineScript",
+        "scriptName": "",
+        "arguments": "-path $(PB_VsoToolsDir)",
```

Comment:
Added quotes.  Left `path` - I think this makes it more clear what we're going to do with the argument.